apps/client/src/pages/builder/sidebars/left/dialogs/profiles.tsx

```diff
@@ -28,6 +30,29 @@ export const ProfilesDialog = () => {
     resolver: zodResolver(formSchema),
   });
 
+  const [iconSrc, setIconSrc] = useDebounceValue("", 400)
+
+  const handleIconChange = useCallback((event: React.ChangeEvent<HTMLInputElement>, field: ControllerRenderProps<{
+    id: string;
```

Comment:
You could re-use the FormValues type here instead of defining the type all over again.